ci/docker/runtime_functions.sh

```diff
@@ -755,6 +755,35 @@ build_ubuntu_gpu_cmake() {
     ninja -v
 }
 
+build_ubuntu_cpu_large_tensor() {
+    set -ex
+    build_ccache_wrappers
+    export CC="ccache gcc"
```

Comment:
build_ccache_wrappers should be enough. My PR also fixes that function: https://github.com/apache/incubator-mxnet/pull/14631

Would be good to merge then no need to export the CC and CXX variables.

Reply:
No longer needed after I change to using cmake